examples/echo.rs

```diff
@@ -64,9 +60,10 @@ fn main() {
 
     let (dht_pk, dht_sk) = gen_keypair();
 
-    let real_sk_bytes: [u8; 32] = FromHex::from_hex(SELF_SK).unwrap();
-    let real_sk = SecretKey::from_slice(&real_sk_bytes).unwrap();
-    let real_pk = real_sk.public_key();
+    // create random tox id and print it
+    let (real_pk, real_sk) = gen_keypair();
+    let id = tox::toxcore::toxid::ToxId::new(real_pk.clone());
+    println!("your tox id is: {:X}\n",id);
```

Comment:
`println` inserts new line at the end itself, so it will print one extra new line because of `\n`.